modules/ROOT/pages/platform/security/secure-connections.adoc

```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
 
 image::azure_privatelink_03_browser_bloom_over_vpn.png["Accessing Browser and Bloom over a VPN", title="Accessing Browser and Bloom over a VPN"]
 
-==== Enabling private endpoints
+==== Enabling Azure private endpoints step-by-step
 
 To enable private endpoints using Azure Private Link:
 
 . Select *Network Access* from the sidebar menu of the Console.
 . Select *New network access configuration* and follow the setup instructions.
 
+*Configure Network Access*
+
+. Choose your product from the available options.
+. Select the appropriate region for your deployment.
+. Enter the "Target Azure Subscription ID" where you want to enable PrivateLink.
+. Enable PrivateLink by toggling the respective option.
+
+*Obtain PrivateLink Service Name*
+
+. After enabling PrivateLink, you will receive a PrivateLink service name.
+. Copy this service name and save it for later use.
+
+*Create PrivateLink Endpoint in the Azure portal*
+
+. Log in to your Azure portal.
+. Navigate to your cloud VPC and create a new PrivateLink endpoint.
+. Use the PrivateLink service name obtained in the previous step for the configuration.
+
+*Accept Endpoint in Aura Console*
+
+. Return to the Aura Virtual Dedicated Cloud Console.
```

Comment:
```suggestion
. Return to the Aura Console.
```
The console is the console, right? VDC is a tier. Unsure of the terminology here.